cabal.project

```diff
@@ -54,3 +54,35 @@ package nothunks
 -- (NOTE this will change to ieee754 in newer versions of nixpkgs).
 extra-packages: ieee, filemanip
 
+if impl(ghc >= 9.6)
+  allow-newer:
+    , *:base
+    , *:ghc-prim
+    , *:mtl
+    , *:template-haskell
+    , *:transformers
+    , hedgehog:*
+    , microlens-mtl:*
+    , protolude:*
+    , th-compat:*
+
+-- Version on Hackages does not compile with `ghc-9.6`
```

Comment:
These can also go inside the `if impl` block, in fact!

Reply:
Done!